lib/optimizely/config_manager/fallback_project_config_manager.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+#
+#    Copyright 2019, Optimizely and contributors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+require_relative 'project_config_manager'
+
+module Optimizely
+  class FallbackProjectConfigManager < ProjectConfigManager
+    # Implementation of ProjectConfigManager interface.
+    # Returns the stored ProjectConfig instance.
+
+    def initialize(datafile = nil, logger = nil, error_handler = nil, skip_json_validation = false)
```

Comment:
Fallback projectconfigmanager should only take `ProjectConfig`. to create DatafileProjectConfig from `datafile` must be in any utils method.